README.md

```diff
@@ -294,6 +289,26 @@ The claim's provider has an attribute 'vat_registered' which governs whether or
 
 For both VAT registered and unregistered LGFS providers, a VAT amount field is provided for manual input of VAT
 
+### Kubernetes
+
+Config for kubernetes object can be found under `kubernetes_deploy/`. In addition binaries in `kubernetes_deploy/bin` have been generated from scripts in `kubernetes_deploy/scripts`.
```

Comment:
I'm not clear on the benefits of `regenerating binaries` but this looks impressive! :)
Are the `*.sh.x.c` part of this too?

Reply:
yes, they are what is generated by `shc`. i was of two minds whether to bother and we can easily remove the binaries and rely on `kubernetes_deploy/scripts/build.sh` instead if the overhead is tedious

the flow for the binaries is just amend the script then run
`shc -f  kubernetes_deploy/script/build.sh -o kubernetes_deploy/bin/build`
to regenerate the binary